Boards/arduino_mega.board.json

```diff
@@ -35,10 +34,12 @@
   ],
   "Info": {
     "CanInstallFirmware": true,
+    "CanResetBoard": true,
```

Comment:
This new property controls whether the reset board menu option is available for the device. Originally the Pico couldn't be reset so I added this in. Pico can be reset now but it seemed worth leaving this in for future use.